### DISCUSSION TITLE
CVE-2022-42920  Security issue detected in latest Spot bug Version

### DISCUSSION THREAD

**Discussion by PrasannaKumarGowdru** [spotbugs/spotbugs-gradle-plugin#805] [Q&A]
Hi Team,

We are consuming 
"com.github.spotbugs" version "5.0.13" latest version in our product and this resulted in security vulnerability with **CVE-2022-42920**  issue.
Please can you tell us when the fix will be available for this issue and when can we expect new version from Spot Bug with this fix.

Thanks,
Prasanna

**Comment by Vampire** [comment]
This project does not use bcel.
Spotbugs is using bcel.
And as soon as a version with the fix that is already committed is released, you can configure this project to use that fixed verision.
See https://github.com/spotbugs/spotbugs/discussions/2251 for the discussion in the right project. ;-)

**Comment by hazendaz** [comment]
issue is not present in recent releases, closing.
